aws/aws_aurora/snapshot/main.tf

```diff
@@ -0,0 +1,53 @@
+# Summary: Create a snapshot of an RDS Aurora MySQL 1 Cluster instance
+
+# Documentation: https://www.terraform.io/docs/language/settings/index.html
+terraform {
+  required_version = ">= 0.14.0"
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 3.38"
+    }
+  }
+}
+
+# Documentation: https://www.terraform.io/docs/language/providers/requirements.html
+provider "aws" {
+  region  = "us-east-1"
+  profile = "terraform-examples"
+  default_tags {
+    tags = {
+      cs_terraform_examples = "aws_aurora"
```

Comment:
This tag should be unique (between all examples) and we typically use the last 2 parts of the directory path here. 

So here it would be: `aws_db_cluster_snapshot/simple` (after the rename mentioned above)